### PR TITLE
Update data share SDK from latest swagger

### PR DESCRIPTION
**Pull request by dadihe**

### CODE REVIEW

**Comment by dadihe** [comment]
swagger change PR is here : https://github.com/Azure/azure-rest-api-specs-pr/pull/971

File: sdk/datashare/Microsoft.Azure.Management.DataShare/src/Generated/Models/ShareSubscription.cs [hunk 1]
```diff
@@ -33,41 +33,50 @@ public ShareSubscription()
...
+        /// <param name="userEmail">Email of the user who created the
+        /// resource</param>
+        /// <param name="userName">Name of the user who created the
+        /// resource</param>
+        public ShareSubscription(string invitationId, string sourceShareLocation, string id = default(string), string name = default(string), string type = default(string), System.DateTime? createdAt = default(System.DateTime?), string providerEmail = default(string), string providerName = default(string), string providerTenantName = default(string), string provisioningState = default(string), string shareDescription = default(string), string shareKind = default(string), string shareName = default(string), string shareSubscriptionStatus = default(string), string shareTerms = default(string), string userEmail = default(string), string userName = default(string))
```
Comment (isra-fel) [inline sdk/datashare/Microsoft.Azure.Management.DataShare/src/Generated/Models/ShareSubscription.cs]: You are making a breaking change to the public api by adding a mandatory parameter. That's why the test fails.

Please consider if this breaking change is what you want. If it is, update the tests and re-record them; if not update swagger spec and re-generate the SDK.

**Review by isra-fel — CHANGES_REQUESTED** [review]

**Comment by isra-fel** [comment]
Hi @dadihe  kindly ping

File: sdk/datashare/Microsoft.Azure.Management.DataShare/src/Generated/Models/ShareSubscription.cs (hunk 1 above)
Comment (dadihe) [reply sdk/datashare/Microsoft.Azure.Management.DataShare/src/Generated/Models/ShareSubscription.cs]: I've reverted and regenerated from the latest swagger for now.

**Comment by dadihe** [comment]
Hi  @isra-fel  can you please review the build error and point out what needs to be fixed?

**Comment by isra-fel** [comment]
@mitchdenny Hi Mitch, what is "net - core - ci" and is it supposed to run in management plane PRs? Thanks

**Comment by isra-fel** [comment]
@dadihe please revert the change to `tools/PS-VSPrompt.lnk`. Other changes look good to me. Thanks

**Comment by dadihe** [comment]
> @dadihe please revert the change to `tools/PS-VSPrompt.lnk`. Other changes look good to me. Thanks

reverted

**Review by isra-fel — APPROVED** [review]

**Pull request merged**